cost/aws/rightsize_redshift/README.md

```diff
@@ -0,0 +1,90 @@
+# AWS Rightsize Redshift
```

Comment:
Should the name of the policy be "AWS Rightsize Redshift Cluster Nodes"?

Reply:
I intentionally left the name ambiguous in case we want to extend it to include changing the number of nodes in addition to their size at some point.